### ISSUE TITLE
Improve landing page

### ISSUE THREAD

**Issue by xrchz** [stakevrun/frontend#345]
The current page has a big bar chart logo which makes it look a bit like a work-in-progress on template. Can we make this something that looks more polished and live? Maybe use the Vrün logo, or something? And make it more appealing+easy to find out more about what this site is about.

**Comment by bpietroniro** [comment]
Is there a `svg` version of the logo on hand?

**Comment by xrchz** [comment]
Yes it's at https://discord.com/channels/1216708060832075806/1216708133628416090/1262108477611049100

**Comment by xrchz** [comment]
Maybe we need to change the responsiveness and size settings so that the logo doesn't show up huge when the window isn't wide enough. Either make the logo go away or get smaller in that case?

**Comment by bpietroniro** [comment]
From Discord:

- Resources box isn't scrollable, and isn't displaying fully - can't get the link to the docs
- Everything on the navbar should be in the same place, connect wallet and dark mode can be on the right
- Can delete the License link, and replace Terms and Conditions with just "Terms" and be a link to a static page that includes a (currently placeholder) render of the terms - HTML from markdown, are add React markdown parser
- Add back the network selector (Holešky vs Mainnet)
- We can remove the "under development" disclaimer eventually - for now remove email address
